addons/hr_recruitment/models/calendar.py

```diff
@@ -8,6 +8,21 @@ class CalendarEvent(models.Model):
     """ Model for Calendar Event """
     _inherit = 'calendar.event'
 
+    @api.model_create_multi
+    def create(self, vals_list):
+        events = super(CalendarEvent, self).create(vals_list)
+        if "default_applicant_id" in self.env.context:
+            applicant_attachments = self.env['ir.attachment'].browse(self.env.context['attachment_ids'])
```

Comment:
Not a good idea. You are allowing anyone to "steal" any attachment based on ids in the context.

Reply:
Hello, ty for catching that, rewrote that piece of code.